sentry-ruby/Gemfile

```
@@ -29,7 +29,7 @@ gem "benchmark_driver"
 gem "benchmark-ipsa"
 gem "benchmark-memory"
 
-gem "yard", github: "lsegal/yard"
+gem "yard", github: "lsegal/yard", branch: "main"
```

I used yard's GH source because we were waiting for a fix (https://github.com/getsentry/sentry-ruby/pull/1994), which has been released now. So I think we can do this instead:

```suggestion
gem "yard"
```